ui/src/views/Rules/Rule/index.jsx

```diff
@@ -170,7 +170,9 @@ function Rule({ isNewRule, user, ...props }) {
     Promise.all([fetchReleaseNames(), fetchReleaseNamesV2()]).then(
       ([names, namesV2]) => {
         if (names.data.data && namesV2.data) {
-          setReleaseNames(names.data.data.names.concat(namesV2.data).sort());
+          setReleaseNames(
+            names.data.data.names.concat(namesV2.data.data).sort()
```

Comment:
3 names and 4 datas in one line, wow! It might be worth doing some of the parsing in `services/releases.js` instead (as we do in the code you're removing) -- but that's not at all a blocker.